.vscode/launch.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
What does this file do? If you don't know it, you should not include it. @cc0441